pkg/image/importer/importer.go

```diff
@@ -50,6 +50,9 @@ type ImageStreamImporter struct {
 	limiter   flowcontrol.RateLimiter
 
 	digestToRepositoryCache map[gocontext.Context]map[manifestKey]*api.Image
```

Comment:
@smarterclayton Can you explain, why is it necessary to group the entries by context? Does the same scenario apply to the `digestToLayerSizeCache` below?